packages/vm/test/api/index.spec.ts

```diff
@@ -225,8 +225,9 @@ describe('VM -> setHardfork, state (deprecated), blockchain', () => {
       const contractAddress = Address.fromString('0x00000000000000000000000000000000000000ff') // contract address
       // setup the vm
       const common = new Common({ chain: Chain.Mainnet, hardfork: Hardfork.Istanbul })
+      const common2 = new Common({ chain: Chain.Mainnet, hardfork: Hardfork.Istanbul })
       const vmNotActivated = await VM.create({ common })
-      const vmActivated = await VM.create({ common, activatePrecompiles: true })
+      const vmActivated = await VM.create({ common: common2, activatePrecompiles: true })
```

Comment:
I'm not sure here, why is the extra `common2` necessary here?

Reply:
Tested, not necessary any more after the latest initialization fix. 🙂 Will remove.